serverr/routes/router.js

```diff
@@ -0,0 +1,126 @@
+var router = require('express').Router();
+var JsonWebToken = require('jsonwebtoken');
+var axios = require('axios');
+var { usersModelColletion, setUserInformation } = require('../userschema');
+var { findUserByEmail, saveUser, upDateUserPassword, deleteUserAccount } = require('../satabase service/databaseservice');
+var { hashUserPassword, comparePassword } = require('../hash password/hashpassword');
+
+var accessPorts = require('../access/access');
+router.use('/', accessPorts);
+
+// console.log(process.env.SECRET_KEY)
+
+// check if user exsist
+var checkUserExsist = async (request, response, next) => {
+    findUserByEmail(request.body.email)
+        .then((user) => {
+            user ? response.send({ message: 'User exsixt' }) : next()
+        }).catch((error) => console.log(error))
+}
+
+
+
+// user register , hash password
+router.post('/register', checkUserExsist, (request, response, next) => {
+    hashUserPassword(request.body.password, 10)
+        .then((hashPassword) => {
+            var newUser = setUserInformation(request.body.email, hashPassword, false, false);
+            saveUser(newUser, response);
+        }).catch((error) => console.log(error))
+})
+
+
+
+// user log in , get token
+router.get('/login/:email/:password', async (request, response) => {
+    var { email, password } = await request.params;
+    findUserByEmail(email)
+        .then((user) => {
+            comparePassword(password, user.password)
+                .then((result) => {
```

Comment:
```suggestion
                .then(result => {
```
unnecessary brackets surrounding single argument function. remove it